pkg/proxy/healthcheck/healthcheck_test.go

```diff
@@ -142,8 +142,11 @@ type fakeProxyHealthChecker struct {
 	healthy bool
 }
 
-func (fake fakeProxyHealthChecker) Health() (bool, time.Time) {
-	return fake.healthy, time.Now()
+func (fake fakeProxyHealthChecker) Health() ProxyHealthCheckStatus {
+	return ProxyHealthCheckStatus{
+		Healthy:     fake.healthy,
+		LastUpdated: time.Now(),
+	}
```

Comment:
If the additional fields aren't relevant to the unit tests then explain why. (Does this mean the unit tests aren't testing some functionality?)